playbooks/system-criticality.md

```diff
@@ -31,21 +31,24 @@ Retired systems, spikes, or time-bound experiments that aren't significantly use
-| Data                         |                                                                              | Weekly backups                                                                                             | Daily backups                                                                                                                                                                     |
-| Security                     |                                                                              | Known vulnerabilities patched within 30 days                                                               | Known vulnerabilities patched within 7 days                                                                                                                                       |
+## System expectations (draft)
+
+Each level has corresponding expectations for how the system is built, operated, and maintained. These are _target_
+expectations and existing systems may not fully comply [yet]. New systems should aim to abide by these as much as
+possible.
+
+|                   | Level 1                                                                                                                                                                                                    | Level 2                                                  | Level 3                                                                                                                                 |
+| ----------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| Development       | READMEs contain up-to-date set-up instructions<br/>Code review on all [pull requests](/playbooks/engineer-workflow.md#pull-requests)<br/>New systems or significant new components undergo [tech review]() | &larr;ditto                                              | &larr;ditto +<br/>Production environment is replicable locally (TODO)                                                                   |
+| Testing           | Automated tests<br/>Tests are run on PRs and `master` by a Continuous Integration pipeline                                                                                                                 | &larr;ditto                                              | &larr;ditto +<br/>Test coverage tooling                                                                                                 |
```

Comment:
Re `Test coverage tooling` do we expect these services to have a target test coverage? Are there generic rules that apply to all our level 3 services such as: must have at least 50% test coverage and no commit can decrease test coverage?

Reply:
We discussed specifying a coverage ratio but decided not to since [at least for the time being] a single standard wouldn't be meaningful given the very different places projects are starting from.
Personally, yeah I'm hope that making coverage more _visible_ will give us some more experience and insight into the most practical additional standards to set, like requiring that commits don't decrease coverage.

Reply:
We currently have a check on Force and Positron to target 70% coverage on a diff. There is a circle check that will fail if the diff is lower, but it does not block merging -- still working to tune this to specific app directories and ignore config files, which I think may confuse our stats. 
<img width="832" alt="Screen Shot 2019-09-11 at 2 24 28 PM" src="https://user-images.githubusercontent.com/1497424/64724181-fc579e80-d49f-11e9-99dd-12565025ecd5.png">